src/modules/form/types.ts

```diff
@@ -82,6 +82,7 @@ export interface DynamicInputCommonProps {
   placeholder?: string;
   breakpoints?: BreakpointOverrides;
   ariaLabelledBy?: string;
+  themeColor?: string;
```

Comment:
Maybe this is an abuse of `DynamicInputCommonProps`, since it's not really common to anything else? This is to allow one signature component to have a blue color and the other to have purple (or whatever the installation's theme primary and secondary colors are), see lines 26-28 of `SignatureEnvelope.tsx`